docs/ref/contrib/gis/gdal.txt

```diff
@@ -611,6 +611,26 @@ coordinate transformation:
         >>> polygon.geom_count
         1
 
+    .. attribute:: has_curve
+
+    .. versionadded:: 5.1
+
+    A boolean indicating if this geometry is or has curve geometry.
```

Comment:
Is the "is or has" distinction really necessary? Could we just say "has"?

Reply:
How about?
```suggestion
    A boolean indicating if this geometry is or contains a curve geometry.
```